comp/trace/compression/fx-zstd/fx.go

```diff
@@ -3,8 +3,8 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-// Package fxzstd provides fx options for the compression component.
-package fxzstd
+// Package fx provides fx options for the compression component.
+package fx
```

Comment:
Looks like we have a discrepancy between the docs and the linter 😄 
https://datadoghq.dev/datadog-agent/components/creating-components/#the-fx-folders-compcompressionfx-zstdfxgo

Could we make sure that linters and documentation matches

Reply:
PR up for documentation update: https://github.com/DataDog/datadog-agent/pull/28072